lib/v2/tongji/radar.js

```diff
@@ -1,6 +1,14 @@
 module.exports = {
     'tongji.edu.cn': {
         _name: '同济大学',
+        bks: [
```

Comment:
https://github.com/DIYgod/RSSHub/blob/b64cf0156066c182fc54b8da5173bc01c60aa38b/lib/v2/tongji/bks.js#L6
```suggestion
        bksy: [
```